nemo/core/classes/modelPT.py

```diff
@@ -1115,6 +1119,49 @@ def cfg(self, cfg):
         self._cfg = cfg
         self._set_hparams(cfg)
```

Comment:
can we save trainer's param's here as well?

Reply:
We dont receive the trainer config inside the constructor, just the Trainer object, so it cant be saved easily. Need to look into it.